.cirrus.yml

```diff
@@ -1,16 +1,20 @@
 environment:
+  HOME: "/root"
   CIRRUS_WORKING_DIR: "/tmp/criu"
-container:
-  # This test case does not work on focal, because the newer version
-  # of libvirt fails during 'vagrant up' with ip6tables errors.
-  image: ubuntu:bionic
-  kvm: true
+
+compute_engine_instance:
+  image_project: cirrus-images
+  image: family/docker-kvm
```

Comment:
Had to use our VM because of [this gotcha](https://github.com/cirruslabs/cirrus-ci-docs/pull/813).

```yaml
image_project: ubuntu-os-cloud
image: family/ubuntu-1804-lts
```

Didn't work because of that licensing issue.